assets/css/admin/downloads/_custom-prices.scss

```diff
@@ -0,0 +1,41 @@
+
+.edd-custom-price-option {
+	&-sections-wrap {
+		display: none;
+		border-width: 0 1px 1px;
+		border-style: solid;
+		border-color: #e5e5e5;
+		box-sizing: border-box;
+		width: 100%;
+	}
+
+	&-section {
+		display: block;
+		padding: 10px 8px;
+		border-bottom: 1px solid rgba( 222, 222, 222, 0.3 );
+
+		&-title {
+			display: block;
+			font-size: 14px;
+			font-weight: 600;
+			padding: 0 0 10px;
+		}
+
+		&-content {
+			display: flex;
+			gap: 12px;
+			margin-bottom: 6px;
+
+			.edd-form-group {
+				display: inline-flex;
+				flex-direction: column;
+				justify-content: flex-end;
+				margin-bottom: 0;
+			}
```

Comment:
Is there a specific use case for this? Not that it will always be avoidable, but any time we need to override or redefine anything related to the [form input markup](https://github.com/easydigitaldownloads/easy-digital-downloads/wiki/Form-Input-Markup) we should ensure there isn't something we can fix on a more global level. 

If not, then changes should be commented inline so we know why it was overwritten.

Reply:
It's actually copying (mostly) from the CSS for the same in [bundled/repeatable rows](https://github.com/easydigitaldownloads/easy-digital-downloads/blob/release/3.0/assets/css/admin/style.scss#L1575-L1582), so I should just get the class added there, I guess.

Reply:
@robincornett I think the issue here is that we still aren't providing a generic enough way for extensions to create similar "columns" in these areas -- instead we are overriding the default behavior of `.edd-form-group` in very specific areas which can cause unpredictability/confusion if you aren't intimately familiar with those two additional classes. 

An extension would need to provide their own CSS or add a wrapper of `.edd-repeatable-row-standard-fields` to their output which isn't something we should encourage.

Instead, expanding our form input markup to include something like `.edd-form-row` that turns `.edd-form-group` children in to columns is something we could document and make predictable. This PR isn't the place for that, but it would be a good time to document why `.edd-repeatable-row-standard-fields .edd-form-group, .edd-bundled-product-row .edd-form-group` needs to exist for the time being.

Reply:
I would be fine to set that up; it totally makes sense to me. I think we actually could argue for putting it in here so we can make sure the linked extensions add the class in their PRs as well.

Reply:
@robincornett Good point -- that will help with the changes required in the extensions when the script is removed. Let's  go ahead and add it here.

Reply:
@spencerfinnell what about just adding a BEM variation class, so the form group class would be `"edd-form-group edd-form-group--row"`? I've added that in the [PR for All Access](https://github.com/easydigitaldownloads/edd-all-access/pull/372), but without styling. Then we could replace the multiple class styling with that, and get the core CSS updated.

Reply:
@robincornett Does that still rely on `.edd-custom-price-option-section-content` being present so the parent has `display: block;`?

I think we should still provide a generic parent:

```html
<div class="edd-form-row">
  <div class="edd-form-group is-column" />
</div>
```

or

```html
<div class="edd-form-row">
  <div class="edd-form-row__column edd-form-group" />
</div>
```

I'm not sure either is more right than the other. If using the variant/modifier class it should just use the `is-` syntax [that we already use](https://github.com/easydigitaldownloads/easy-digital-downloads/blob/release/3.0/assets/css/admin/style.scss#L4074-L4077) and is recommended by the [WordPress CSS coding standards](https://developer.wordpress.org/block-editor/contributors/develop/coding-guidelines/#naming)

Reply:
I've opted for the first: `edd-form-row` and `edd-form-group is-column`. Started a "forms" directory for these classes but so far have only put the rows in.

If we want to go this route, we should update the wiki and add a new issue to update the core markup as needed.